app/src/main/java/org/listenbrainz/android/repository/AppPreferences.kt

```diff
@@ -37,8 +37,21 @@ interface AppPreferences {
     var currentPlayable : Playable?
     
     /* Login related preferences */
+    
     val loginStatus: Int
-    val accessToken: String?
+    /****MusicBrainz User Token:** Obtained when user logins in.*/
+    val mbAccessToken: String?
```

Comment:
Need to remove this in the upcoming PRs.

Reply:
I added this since feed is going to require lbAccessToken. Shouldn't this naming fit in? Just wondering ☺️